src/Expression.php

```diff
@@ -376,17 +358,12 @@ protected function _escape($value)
      * query rendering. Consider using `_consume()` instead, which will
      * also handle nested expressions properly.
      *
-     * @param string|array $value String literal or array of strings containing input data
+     * @param string|int|float $value
```

Comment:
I guess it should be `mixed` type here.

Reply:
mixed means any, I think it should be string or scalar (we should cast everything to string in dsql anyway...)

Reply:
![image](https://user-images.githubusercontent.com/2228672/86132939-e925c600-bae7-11ea-84df-d1ae6c4db58e.png)